### PR TITLE
Convert procedure to list, shorten texts, workaround for missing paragraphs

### PR DESCRIPTION
**Pull request by cadamini**
### Description

Shorten texts in the article (after investigating the issue below) - I decided to add a workaround for missing paragraphs within the list.

### Motivation

This makes the content easier to read, understand, and digest. I love writing.

### Related issues and pull requests

Relates to https://github.com/mdn/yari/issues/5042 - I originally created the PR to investigate the bug reported a year ago.

### CODE REVIEW

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 1]
```diff
@@ -167,11 +168,11 @@ Let's have a go at writing our own functions featuring return values.
    });
    ```
 
-   Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
+    Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:171]:
```suggestion
    Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 2]
```diff
@@ -167,11 +168,11 @@ Let's have a go at writing our own functions featuring return values.
...
+    Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.  
 
-   Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
+    Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:173]:
```suggestion
    Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 3]
```diff
@@ -167,11 +168,11 @@ Let's have a go at writing our own functions featuring return values.
    });
    ```
 
-   Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
+    Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:171]:
```suggestion
   Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 4]
```diff
@@ -167,11 +168,11 @@ Let's have a go at writing our own functions featuring return values.
...
 
-   Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
+    Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:173]:
```suggestion
   Next, we do a conditional test. If the entered value is not a number, an error message is printed on the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 5]
```diff
@@ -167,11 +168,11 @@ Let's have a go at writing our own functions featuring return values.
...
 
-   If the test returns `false`, the `num` value is a number. Therefore, a sentence is printed out inside the paragraph element that states the square, cube, and factorial values of the number. The sentence calls the `squared()`, `cubed()`, and `factorial()` functions to calculate the required values.
+    If the test returns `false`, the `num` value is a number. Therefore, a sentence is printed out inside the paragraph element that states the square, cube, and factorial values of the number. The sentence calls the `squared()`, `cubed()`, and `factorial()` functions to calculate the required values.
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:175]:
```suggestion
   If the test returns `false`, the `num` value is a number. Therefore, a sentence is printed out inside the paragraph element that states the square, cube, and factorial values of the number. The sentence calls the `squared()`, `cubed()`, and `factorial()` functions to calculate the required values.
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 6]
```diff
@@ -152,7 +153,8 @@ Let's have a go at writing our own functions featuring return values.
 
    The `squared()` and `cubed()` functions are fairly obvious — they return the square or cube of the number that was given as a parameter. The `factorial()` function returns the [factorial](https://en.wikipedia.org/wiki/Factorial) of the given number.
 
-3. Next, we're going to include a way to print out information about the number entered into the text input. Enter the following event handler below the existing functions:
+3. Include a way to print out information about the number entered into the text input by adding the following event handler below the existing functions:
+
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:157]:
```suggestion
```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 7]
```diff
@@ -166,16 +167,16 @@ Let's have a go at writing our own functions featuring return values.
      }
    });
    ```
+
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:170]:
```suggestion

```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 8]
```diff
@@ -166,16 +167,15 @@ Let's have a go at writing our own functions featuring return values.
      }
    });
    ```
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:169]:
```suggestion
   ```

```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 9]
```diff
@@ -165,17 +166,15 @@ Let's have a go at writing our own functions featuring return values.
        para.textContent += `${num} factorial is ${factorial(num)}. `;
      }
    });
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:168]:
```suggestion
   });
```

```

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 10]
```diff
@@ -165,17 +166,17 @@ Let's have a go at writing our own functions featuring return values.
...
-   ```
-
-   Here we are adding a listener to the `change` event. It runs whenever the `change` event fires on the text input — that is, when a new value is entered into the text `input`, and submitted (e.g., enter a value, then un-focus the input by pressing <kbd>Tab</kbd> or <kbd>Return</kbd>). When this anonymous function runs, the value in the `input` is stored in the `num` constant.
+```
```
Comment (cadamini) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:169]:
````suggestion
```

````

**Comment by cadamini** [comment]
I gave up fixing the linter.

File: files/en-us/learn/javascript/building_blocks/return_values/index.md [hunk 11]
```diff
@@ -167,15 +168,15 @@ Let's have a go at writing our own functions featuring return values.
...
-
-   Next, we do a conditional test. If the entered value is not a number, an error message is printed to the paragraph. The test looks at whether the expression `isNaN(num)` returns `true`. The [`isNaN()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) function tests whether the `num` value is not a number — if so, it returns `true`, and if not, it returns `false`.
+4. Save your code, load it in a browser, and try it out.
 
-   If the test returns `false`, the `num` value is a number. Therefore, a sentence is printed out inside the paragraph element that states the square, cube, and factorial values of the number. The sentence calls the `squared()`, `cubed()`, and `factorial()` functions to calculate the required values.
+Here are some explanations for the `addEventListener` function in step 3 above:
```
Comment (wbamberg) [inline files/en-us/learn/javascript/building_blocks/return_values/index.md:173]:
```suggestion
Here are some explanations for the `addEventListener` function in step 3 above:
```

**Review by wbamberg — APPROVED** [review]
👍 thank you for this PR, @cadamini , and sorry to be so slow to review it. Looks great!

**Pull request merged**